docs/shrc.md

```diff
@@ -260,6 +260,7 @@ permission com.ibm.oti.shared.SharedClassesNamedPermission "getSharedCacheInfo";
 permission com.ibm.oti.shared.SharedClassesNamedPermission "destroySharedCache";
 ```
 
+From OpenJDK version 18 onwards, if you enable the `SecurityManager` at runtime by calling the `System.setSecurityManager()` API, you must set the `-Djava.security.manager=allow` option. To disable the `SecurityManager`, specify the `-Djava.security.manager=disallow` option. If an application is designed to run on multiple OpenJDK versions, the same command line might be used across multiple versions. OpenJDK versions before version 12 now ignore the `allow` and `disallow` keywords, if specified.
```

Comment:
This has nothing to do with shared classes, not sure why it's added here.

Reply:
SecurityManager is discussed in this topic (https://eclipse.dev/openj9/docs/shrc/#support-for-custom-class-loaders). So I figured we can add this as an additional information. The other place where SecurityManager is mentioned is in https://eclipse.dev/openj9/docs/xxenablecriusupport/. I was not sure that this information will fit in there. Ideally, I would have preferred to have a topic for `Djava.security.manager` system property where I could have added this new change but there is no such topic.